sample.env

```diff
@@ -132,6 +132,9 @@ SMTP_STARTTLS_AUTO=
 # Specify the email address that all mail is sent from
 SMTP_SENDER=
 
+# Comment this out if you are running into mailserver certificate issues
```

Comment:
This comment is going to need to be more descriptive than this. Should probably include this as part of the other SMTP settings instead

Reply:
so i would be moving this below the starttls auto, but also give it its own comment. As this shouldn't be enabled by accident. Should the comment include a security notice / warning?

Reply:
I think the way you have it is fine. I can fix up the wording and formatting in a separate PR. Thank you :)